mvc/src/main/java/nextstep/mvc/view/JsonView.java

```diff
@@ -1,13 +1,41 @@
 package nextstep.mvc.view;
 
+import static nextstep.web.support.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-
 import java.util.Map;
 
 public class JsonView implements View {
 
+    private final ObjectMapper objectMapper;
+
+    public JsonView() {
+        this.objectMapper = new ObjectMapper();
+    }
+
     @Override
     public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response) throws Exception {
+        response.setContentType(APPLICATION_JSON_UTF8_VALUE);
+        final String body = getBody(model);
+        response.getWriter().write(body);
+    }
+
+    private String getBody(final Map<String, ?> model) throws JsonProcessingException {
+        if (model.size() == 1) {
+            final Object attribute = getAttribute(model);
+            return objectMapper.writeValueAsString(attribute);
+        }
```

Comment:
안녕하세요 디우! 

objectMapper 사용 방법을 새롭게 익히느라 힘드셨을 것 같아요.

해당 로직은 `데이터가 하나일 경우`에 데이터 값 그대로 반환해주려고 작성하신 것으로 보이는데

value의 개수를 아는 상태에서 스트림을  사용해서 데이터를 가져오는 로직이 조금 무거워보여요! 

아래와 같은 로직으로 스트림 없이 추출할 수도 있을 것 같아요 ☺️
```suggestion
          if (model.size() == 1) {
            final String key = (String) model.keySet().toArray()[0];
            final Object value = model.get(key);
            objectMapper.writeValue(writer, value);
            return;
        }
```

```

Reply:
오호..!! 그러네요 이미 size를 아는데, 굳이 스트림을 돌면서 가져오는건 낭비였네요..😅
추가적으로 코드 개선해보았습니다!!
또한 `writerWithDefaultPrettyPrinter()` 부분도 제거해보았는데, 저희 팀의 경우 프런트와 실제로 네트워크를 타고 통신할 때 공백을 추가해주지 않는다고 하더라구요..불필요한 비용이 추가되는 것 같아서 제거해보았습니다!!